CRAMv3.tex

```diff
@@ -1093,8 +1093,8 @@ \subsection{\textbf{CRAM positional data}}
   \State $reference\_id\gets slice\_header.reference\_sequence\_id$
 \EndIf
 \State $read\_length \gets$ \Call{ReadItem}{RL, Integer}
-\If{$container\_pmap.AP\_delta \ne 0$}
-    \State Initialise $last\_position$ to 0 if first record in slice
+\If{$container\_pmap.AP\_delta \ne 0$ \textbf{and} $slice\_header.reference\_sequence\_id \geq 0$}
+    \State Initialise $last\_position$ to $slice\_header.alignment\_start$ if first record in slice
```

Comment:
Might as well use pseudocode, otherwise LGTM:
```suggestion
    \If{$first\_record\_in\_slice$}
        \State $last\_position\gets$ $slice\_header.alignment\_start$
    \EndIf
```